templates/form/card.liquid

```diff
@@ -81,27 +200,39 @@
           <div class="col-xs-12">
             {% capture button_text %}
               {% if model.transaction_type == "payment" %}
-                {% t Pay %} <span id="total-field">{{ model.formatted_amount }} {{ model.currency }}</span>
+                {% t Pay %}
+                <span id="total-field">{{ model.formatted_amount }} {{ model.currency }}</span>
               {% elsif model.transaction_type == "subscription" %}
                 {% t Create subscription %}
                 {% if config.enable_amount_for_subscription %}
-                  <span id="total-field"> ({{ model.formatted_amount }} {{ model.currency }})</span>
+                  <span id="total-field">
+                    ({{ model.formatted_amount }} {{ model.currency }})</span>
                 {% endif %}
               {% elsif model.transaction_type == "card" %}
                 {% t Save card %}
               {% elsif model.transaction_type == "payout" %}
-                {% t Receive %} <span id="total-field">{{ model.formatted_amount }} {{ model.currency }}</span>
+                {% t Receive %}
+                <span id="total-field">{{ model.formatted_amount }} {{ model.currency }}</span>
               {% endif %}
             {% endcapture %}
-            <button type="submit" class="btn btn-info" data-card-valid-swap="btn-success,btn-info">{{ button_text }}</button>
+            <button
+              type="submit"
+              class="btn btn-info"
+              data-card-valid-swap="btn-success,btn-info">{{ button_text }}</button>
             {% if model.calculate_fee %}
               <p id="fee" class="text-center">
-                {% t Payment fee %}: <span id="fee-field"><i>{% t Fill in card number %}</i></span>
+                {% t Payment fee %}:
+                <span id="fee-field">
+                  <i>{% t Fill in card number %}</i>
+                </span>
               </p>
             {% endif %}
+            {% if config.reservation_notice %}
+              <div class="notice">{{ config.reservation_notice }}</div>
```

Comment:
`.notice` might a little vague, so is maybe already used by some custom brandings. Can we call it `.reservation_notice` instead?